crypto/signer_test.go

```diff
@@ -51,30 +50,29 @@ func (i usesSHA256Hasher) String() string {
 }
 
 func TestSigner(t *testing.T) {
```

Comment:
Would you mind keeping the old test, in addition to the one you added? I think both add value in different areas.